src/event.js

```diff
@@ -1,27 +1,43 @@
 import { isTouch } from './utils'
 
 export const customEvents = popup => {
-	const onMouseLeave = e => {
+
+	var eventListenerStack = [],
```

Comment:
`let` instead of `var`?

Reply:
errr, right i don't know why it happens to `var` here, on it